templates/zerver/emails/base.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
Let's call this file email_base.html, since we already have a base.html in templates/zerver.